tests/appsec/iast/test_iast.py

```diff
@@ -42,6 +42,7 @@ def __expected_weak_cipher_algorithm(self):
         expected = self.EXPECTATIONS.get(context.library.library)
         return expected.get("WEAK_CIPHER_ALGORITHM") if expected else None
 
+    @missing_feature(context.weblog_variant == "spring-boot-openliberty")
```

Comment:
```suggestion
    @bug(context.weblog_variant == "spring-boot-openliberty")
```

(This is a bug, maybe in the tests, rather than missing feature, probably in the way we setup the openliberty docker container)

Reply:
It might be fixed by this: https://github.com/DataDog/system-tests/pull/656